src/components/Detail.js

```diff
@@ -0,0 +1,37 @@
+import React, {Component} from "react";
+
+const imageUrl = "images/peter-wendt--r5KSMkyoSc-unsplash.jpg";
+
+export default class Details extends Component {
+
+    render() {
+        const {blocks} = this.props;
+        const {city, name, id} = this.props.match.params;
+
+        const block = blocks.filter(block => block.name.toLowerCase() === name.toLowerCase())[0];
```

Comment:
what will happened if we won't find such a block? (the exception will be thrown)
PS why not `find` ?